DOCS.md

```diff
@@ -10,6 +10,9 @@ The following parameters are used to configure the plugin:
 * **recipient** - alternatively you can send it to a specific user
 * **username** - choose the username this integration will post as
 * **template** - overwrite the default message template
+* **image** - A valid URL to an image file that will be displayed inside a message attachment
```

Comment:
`image` cannot be used because it is a reserved field in the yaml. It is used to provide the name of the plugin:

```
notify:
    image: plugins/slack
    image_url: http://...
```

the use of `PLUGIN_IMAGE_URL` is correct, we just need to update the documents to use `image_url`